src/python/ensembl/utils/rloader.py

```diff
@@ -0,0 +1,71 @@
+# See the NOTICE file distributed with this work for additional information
+#   regarding copyright ownership.
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#       http://www.apache.org/licenses/LICENSE-2.0
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""
+Allow to seamlessly load / read the content of a remote file as if it was located locally.
+
+@author: Marc Chakiachvili
+
+"""
+import configparser
+import json
+import logging
+from io import StringIO
+
+import dotenv
+import requests
+import requests.exceptions as exc
+import yaml
+
+logger = logging.getLogger(__name__)
+
+
+class RemoteFileLoader:
+    available_format = ['yaml', 'ini', 'env', 'json']
```

Comment:
Now that's not written, makes sense to use a set IMO (although list or tuple aren't wrong). Also I'd use the plural of the name.
```suggestion
    available_formats = {'yaml', 'ini', 'env', 'json'}
```